### ISSUE TITLE
CSS :focus, do we really need it ?

### ISSUE THREAD

**Issue by Choubakawa** [techno-tim/littlelink-server#29]
I may be missing the point of `:focus` (accessibility ?) but I think using this attribute is not needed in this project.

I take the button list like a menu, if I click on it I leave the page and if I return on my littlelink page the button is focused, I would expect the button come back to its initial state.

**Comment by palepinkdot** [comment]
It does remove redundant functionality. the `:focus` and `:hover` pseudo-classes perform duplicate functions. Hovering over an icon darkens the background colors and clicking it, using the `:focus` class forces the button to stay darkened. 

I think for the normal use case for this app, it doesn't really matter `(Open page in embedded mobile browser > Click link > Close embedded browser)`, but it does make things a bit messy with caching the focused result upon returning to the page. Definitely nitpicking with this issue but it seems like it could improve the UX of at least one user, IMO it's worth the change.
